### ISSUE TITLE
Split base16-builder cli from core

### ISSUE THREAD

**Issue by aloisdg** [base16-builder/base16-builder#50]
Hello,

What would you think to split base16-builder in two projects. One for the core and one for the CLI. If someone, someday, want to create a GUI version or another tool. He could use the core part of base16-builer. In C#, i would ask for switching for one project to a group of projects (a PCL, a CLI, why not a wpf, etc).

Basically, can we add more [modularity](https://en.wikipedia.org/wiki/Modular_programming) in base16-builder?

**Comment by alexbooker** [comment]
`base16-builder` _already_ modular. It isn't documented, but it's possible to install `base16-builder` locally:

```bash
npm install base16-builder
```

... And then reference `buildTheme`:

```javascript
import { buildTheme } from 'base16-builder'

buildTheme(...)
```

**Comment by aloisdg** [comment]
Ok. I learnt something.

**Comment by alexbooker** [comment]
Closed by 5aa25ae3a5c573cce1943caa1da6f18a42ce01c6. It's not ideal to reference a ticket as documentation but it's good enough for now.
